engine/classes/Elgg/Application.php

```diff
@@ -267,15 +266,18 @@ public function bootCore() {
 					}
 				}
 
-				_elgg_services()->views->registerPluginViews(Directory\Local::root()->getPath());
+				_elgg_services()->views->registerPluginViews($root->getPath());
 			}
 			
 			if (!elgg_get_config('i18n_loaded_from_cache')) {
-				_elgg_services()->translator->registerPluginTranslations(Directory\Local::root()->getPath());
+				_elgg_services()->translator->registerPluginTranslations($root->getPath());
 			}
 			
 			// This is root directory start.php, not elgg/engine/start.php
-			@include_once Directory\Local::root()->getPath("start.php");
+			$root_start = $root->getPath("start.php");
+			if (is_file($root_start)) {
+				include $root_start;
```

Comment:
This will also not swallow an include error due to permissions (a good thing). Actually shouldn't this be `require`?